.env.example

```diff
@@ -0,0 +1,49 @@
+# DB image settings
```

Comment:
let's put all the testing environment files in a dedicated directory and not in the root of the repo

Reply:
fixed [here](https://github.com/datopian/ckan-ng-harvest/commit/cb767e1195662a6cb9800cbe7e8701df2cacf261)

Reply:
Seems that this file is still here.

Reply:
Moved [here](https://github.com/datopian/ckan-ng-harvest/commit/bf53ca28cabc73a7662770c5e148ac691185170a).
I moved to [docker-extras](https://github.com/datopian/ckan-ng-harvest/tree/66_add_CKAN_container/docker-extras) folder what I think is extra.
Let me know what else do you think we need to move.